osutil/testhelper.go

```diff
@@ -38,3 +42,93 @@ func MustBeTestBinary(panicMsg string) {
 		panic(panicMsg)
 	}
 }
+
+var injectSysroot = "/"
+var stderr io.Writer = os.Stderr
+var foreverLoop = func() {
+	for {
+	}
+}
+
+func injectFault(tagKind string) (injected bool) {
```

Comment:
The result is never checked. I'm also not sure if it should be logged by MaybeInjectFault since even a `false` is a valid result. I think it might more appropriate to return the errors and log those. That way if the tester's test doesn't inject the fault, the failure isn't silently absorbed. Wdyt?